04. 완전탐색(블루투포스)/04. 졸업선물/이서인/index.js

```diff
@@ -1,7 +1,33 @@
 function solution(m, product) {
-  let answer = 0;
+  let sums = [];
 
-  return answer;
+  for (let i = 0; i < product.length; i++) {
+    sums.push(product[i][0] + product[i][1]);
+  }
+
+  sums = sums.sort((a, b) => a - b);
```

Comment:
사실 sort는 원본 배열을 변경해서 할당하지 않아도 변경되어있을거 같아요
```suggestion
  sums.sort((a, b) => a - b);
```